assets/src/helpers.js

```diff
@@ -18,6 +18,8 @@ import { getColorClassName, getColorObjectByAttributeValues, RichText } from '@w
 /**
  * Internal dependencies
  */
+// eslint-disable-next-line no-unused-vars
+import store from './stores/amp-story';
```

Comment:
Importing this to ensure the store is registered before using it. Importing it within `amp-story-editor-blocks.js` even before importing the helpers doesn't seem to be enough.